Alcuin_XML.xml

```diff
@@ -169,911 +169,6922 @@
-          menescire. Quid
-          <choice>
-            <orig>su<macron/></orig>
-            <reg>sum</reg>
-          </choice>
-          ego nisi anima & caro?
-          <lb n="7" />
-          Caro quid sit
-          <choice>
-            <orig>omnib;</orig>
-            <reg>omnibus</reg>
-          </choice>
-          <choice>
-            <orig>notu<macron/></orig>
-            <reg>notum</reg>
-          </choice>
-          est,
-          <choice>
-            <orig>qse</orig>
-            <reg>qui</reg>
-          </choice>
-          sciunt homines
-          <choice>
-            <orig>e<macron/>e<macron/>;</orig>
-            <reg>esse;</reg>
-          </choice>
-          <choice>
-            <orig>anime<macron/></orig>
-            <reg>animae</reg>
-          </choice>
-          <choice>
-            <orig>u<macron/></orig>
-            <reg>vero</reg>
-          </choice>
-          <choice>
-            <orig>ratione<macron/></orig>
-            <reg>rationem</reg>
-          </choice>
-          <choice>
-            <orig>uix</orig>
-            <reg>vix</reg>
-          </choice>
-          <choice>
-            <orig>paucoꝜ</orig>
-            <reg>paucorum</reg>
-          </choice>
-          <lb n="8" />
-          est
-          <choice>
-            <orig>plenit<macron/></orig>
-            <reg>pleniter</reg>
-          </choice>
-          nosse. Nec aliquid magis homini in hac mortalitate vivente necessa-
-          <lb n="9" />
-          -rium est nosse;
-          <choice>
-            <orig>qua<macron/></orig>
-            <reg>quam</reg>
-          </choice>
-          <choice>
-            <orig>D<macron/>m</orig>
-            <reg>Deum</reg>
-          </choice>
-          <choice>
-            <orig>&amp;</orig>
-            <reg>et</reg>
-          </choice>
-          <choice>
-            <orig>anima<macron/>.</orig>
-            <reg>animam</reg>
-          </choice>
-          <choice>
-            <orig>Quantu<macron/></orig>
-            <reg>Quantum</reg>
-          </choice>
-          <choice>
-            <orig>eni<macron/></orig>
-            <reg>enim</reg>
-          </choice>
-          <choice>
-            <orig>quisq:</orig>
-            <reg>quisque</reg>
-          </choice>
-          <choice>
-            <orig>Dm<macron/></orig>
-            <reg>Deum</reg>
-          </choice>
-          agnoscit, in tantum
-          <lb n="10" />
-          diligit. Qui minus agnoscit,
-          <choice>
-            <orig>min<macron/></orig>
-            <reg>minus</reg>
-          </choice>
-          diligit. Ergo naturale est omnihomini
-          <lb n="11" />
-          <choice>
-            <orig>dm<macron/></orig>
-            <reg>deum</reg>
-          </choice>
-          amare. Si naturale
-          <choice>
-            <orig>e<macron/></orig>
-            <reg>est</reg>
-          </choice>
-          homini
-          <choice>
-            <abbr>bonu</abbr>
```

Comment:
I noticed these are inconsistent across our work - can we meet up to decide what the standard is for abbreviations?